### PR TITLE
Quick favor

### PR DESCRIPTION
**Pull request by sharabash**

### CODE REVIEW

File: app/models/quran/word.rb [hunk 1]
```diff
@@ -34,7 +34,12 @@ class Quran::Word < ActiveRecord::Base
   def as_json(options = {})
     super().merge(
       translation: translation.value,
-      transliteration: transliteration.value
+      transliteration: transliteration.value,
+      corpus: {
+        description: corpus.description,
+        image_src: corpus.image_src,
+        segment: corpus.segment
```
Comment (mmahalwy) [inline app/models/quran/word.rb]: `corpus.` will query each time no?

Comment (sharabash) [inline app/models/quran/word.rb]: I dunno, you know how to make it cache?

**Comment by sharabash** [comment]
what exactly is needed on this branch? does it not join? also, if it queries corpus multiple times then its also querying translation/transliteration multiple times, right? since their relationships are the same...

**Comment by sharabash** [comment]
and the answer is no, it will not query corpus multiple times; rails keeps the object around after its loaded the first time, proof:

```
[12] quran_api >  word=Quran::Word.find(2)
 [Debug]     Quran::Word Load (0.4ms)  SELECT  "quran"."word".* FROM "quran"."word" WHERE "quran"."word"."word_id" = $1 LIMIT 1  [["word_id", 2]] (pid:30645)
=> #<Quran::Word:0x000000082b6800> {
     :word_id => 2,
    :ayah_key => "1:1",
    :position => 2,
    :token_id => 10001
}
[13] quran_api >  word.corpus.segment
 [Debug]     Quran::WordCorpus Load (0.4ms)  SELECT  "word_corpus".* FROM "word_corpus" WHERE "word_corpus"."word_id" = $1 LIMIT 1  [["word_id", 2]] (pid:30645)
=> {
    "PN" => {
         "class_name" => "blue",
        "description" => "genitive proper noun",
            "concept" => "Allah",
        "translation" => "Allah",
             "prefix" => "",
            "grammar" => "لفظ الجلالة مجرور"
    }
}
[14] quran_api >  word.corpus.description
=> "The second word of verse (1:1) is a proper noun in the genitive case (مجرور). The proper noun's triliteral root is hamza lām hā (أ ل ه)."
[15] quran_api >  word.corpus.image_src
=> "/wordimage?id=2"

```

notice, it only selected from word_corpus once

**Pull request merged**